### PR TITLE
CAT-1529 Grouped list functions together in FE

### PR DESCRIPTION
**Pull request by Fasusu**
CatroidCheck: https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/CatroidCheck/48/
SingleClassTest: https://jenkins.catrob.at/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/720/

Second commit contains only auto format changes where spaces are replaced with tabs and a newline is added to the end of the file, so that checkstyle does not complain anymore

### CODE REVIEW

**Comment by thmq** [comment]
ok to test

**Comment by thmq** [comment]
could you please format your code according to the bot's suggestions?

**Comment by fwinkelbauer** [comment]
Or just use the Android Studio Formatter like it is mentioned in the first bullet point [here](https://github.com/Catrobat/Catroid/wiki/Creating-a-pull-request)

File: catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java [hunk 1]
```diff
@@ -135,6 +136,23 @@
 	private String actionBarTitle;
 	private int[] itemsIds;
 
+	public FormulaEditorListFragment() {
```
Comment (cheidenreich) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java]: the default constructor would be autogenerated by the compiler -> can be removed

**Comment by thmq** [comment]
this is replaced by #1660.

**Pull request closed without merging**